easybuild/cray_external_modules_metadata-20.06.cfg

```diff
@@ -0,0 +1,75 @@
+# metadata useful to EasyBuild (hpcugent.github.io/easybuild) for modules provided by Cray on Cray systems
+# authors: Guilherme Peretti-Pezzi (CSCS), Petar Forai (IMP/IMBA), Kenneth Hoste (HPC-UGent), Eirini Koutsaniti (CSCS)
+
+[cray-fftw]
+name = FFTW
+
+[cray-hdf5]
+name = HDF5
+version = 1.10.6.1
+prefix = HDF5_DIR
+
+[cray-hdf5-parallel]
+name = HDF5
+version = 1.10.6.1
+prefix = HDF5_DIR
+
+[cray-libsci]
+name = LibSci
+version = 20.06.1
+prefix = CRAY_LIBSCI_PREFIX
+
+[cray-netcdf]
+name = netCDF, netCDF-Fortran
+version = 4.7.3.3, 4.7.3.3
+prefix = NETCDF_DIR
+
+[cray-netcdf-hdf5parallel]
+name = netCDF, netCDF-Fortran
+version = 4.7.3.3, 4.7.3.3
+prefix = NETCDF_DIR
+
+[cray-petsc]
+name = PETSc
+
+[cray-petsc-64]
+name = PETSc
+
+[cray-petsc-complex]
+name = PETSc
+
+[cray-petsc-complex-64]
+name = PETSc
+
+[cray-python]
+name = Python
+version = 3.8.2.1
+prefix = CRAY_PYTHON_PREFIX
```

Comment:
```suggestion
```
Is the version and prefix necessary? Since the module has both the variables `CRAY_PYTHON_VERSION` and `CRAY_PYTHON_PREFIX` Easybuild should be able to get those variables from the module

Reply:
Thanks @ekouts, I just added them to see if they were fixing issues with Python recipes that failed. 
If EasyBuild can automatically detect both values, we can remove them.